front/components/error_boundary/ErrorBoundary.tsx

```diff
@@ -24,12 +24,10 @@ export class ErrorBoundary extends React.Component<
   }
 
   componentDidCatch(error: Error, info: React.ErrorInfo) {
-    datadogLogs.logger.error("Uncaught client side error", {
-      error: {
-        message: error.message,
-        stack: error.stack,
-        name: error.name,
-      },
+    // TODO (yuka): check with Ben what is the best way to log this error
+    console.error(error);
```

Comment:
As discussed [here](https://github.com/dust-tt/dust/pull/15512#issuecomment-3236757281). I will see if this goes to Error Tracking, if it doesn't I will revert them and work on this with Ben next monday.

Reply:
By looking throughout the code I see quite a few import logger from "@app/logger/logger"; in tsx components though